### PR TITLE
(feature/Query_Component)-added query component

### LINKED ISSUE #56: Create codemap_details/Query component
Find the Query.js file in src/codemap_details/Components and write your code in it.
Find the Query.module.css file in src/codemap_details/CSS and write the corresponding CSS there.

Guidelines:
1. Use react-bootstrap
2. The component should be responsive with all screen sizes.
3. Don't use element selector in CSS, use classes instead.

Refer Design here: https://xd.adobe.com/view/1cf18476-27a4-4007-5fc7-0154fbe60af6-6ac0/screen/1b92180c-29d0-436d-b2f7-175dd976fa45/

Query Component

### PR DESCRIPTION
**Pull request by prachibansal2000**
# Description

feature/Query_component-added query component
Fixed #56 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/codemap_details/CSS/Query.module.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.head {
+  margin-top: 50px;
+  letter-spacing: 0px;
+  font-size: 2.3rem;
+  font-weight: 390;
+  font-family: Arial;
```
Comment (vinitshahdeo) [inline src/codemap_details/CSS/Query.module.css]: @prachibansal2000 Are we using Arial font for other pages?

PS: @Manvityagi If yes, let's come up with some awesome font. Arial is too mainstream.

File: src/codemap_details/Components/Query.js [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
+              </p>
+              <hr className={styles.line} />
+            </Col>
+            <Col lg={6}>
+              <Image src={query} className={styles.img} fluid />
+            </Col>
+          </Row>
+        </Container>
+        <Container className={styles.small} fluid>
+          <h1 className={styles.head} style={{ textAlign: "center" }}>
```
Comment (vinitshahdeo) [inline src/codemap_details/Components/Query.js]: @prachibansal2000 Please avoid inline styling. It creates troubles while we scale up.

File: src/codemap_details/Components/Query.js [hunk 3]
```diff
@@ -1,6 +1,56 @@
...
+          <Image
+            className={styles.img}
+            style={{ maxWidth: "90%" }}
+            src={query}
+            fluid
+          />
+          <p className={styles.para} fluid>
+            Duis aute irure dolor in reprehenderit in voluptate velit esse
+            cillum dolore eu fugiat nulla pariatur.
+            <br />
```
Comment (vinitshahdeo) [inline src/codemap_details/Components/Query.js:41]: @prachibansal2000 Why are we using multiple `<br />` tags? Can't we have `margin-bottom`?

**Review by vinitshahdeo — COMMENTED** [review]
@prachibansal2000 Added a few comments, please do have a look.

File: src/codemap_details/Components/Query.js (hunk 2 above)
Comment (prachibansal2000) [reply src/codemap_details/Components/Query.js]: ok

File: src/codemap_details/Components/Query.js (hunk 3 above)
Comment (prachibansal2000) [reply src/codemap_details/Components/Query.js:41]: i have added `<br/>` tags in between the text.

File: src/codemap_details/CSS/Query.module.css (hunk 1 above)
Comment (Manvityagi) [reply src/codemap_details/CSS/Query.module.css]: @aayush-05 Please check this font thing and give a common font to everyone, lets discuss this on the group.

Comment (aa0501) [reply src/codemap_details/CSS/Query.module.css]: Moved to group

File: src/codemap_details/Components/Query.js (hunk 3 above)
Comment (aa0501) [reply src/codemap_details/Components/Query.js:41]: Better to use `<p>` tags imo

Comment (vinitshahdeo) [reply src/codemap_details/Components/Query.js:41]: @prachibansal2000 What about `<span>` in between text? And span will have `margin-bottom`?

`<p>` tag adds a single line break by default but I can see you've put twice.

Comment (prachibansal2000) [reply src/codemap_details/Components/Query.js:41]: i have removed all the `<br/>` tags

**Review by vaishali614 — APPROVED** [review]

**Pull request closed without merging**